i18n/testdata/translation_mismatches.noargs.es.po

```diff
@@ -0,0 +1,41 @@
+# Testing
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"POT-Creation-Date: 2020-03-25 13:57+0000\n"
+"Language: es\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+
+#: Translated/43f7e69e-727d-4cfe-81b8-564e7833052b/name:0
+#: Translated/e42deebf-90fa-4636-81cb-d247a3d3ba75/quick_replies:1
+msgid "Blue"
+msgstr "Azul"
+
+#: Translated/3a044264-81d1-4ba7-882a-e98740c8e724/name:0
+msgid "Other"
+msgstr "Otro"
+
+#: Translated/e42deebf-90fa-4636-81cb-d247a3d3ba75/quick_replies:0
+msgctxt "e42deebf-90fa-4636-81cb-d247a3d3ba75/quick_replies:0"
+msgid "Red"
+msgstr "Rojo"
+
+#: Translated/d1ce3c92-7025-4607-a910-444361a6b9b3/name:0
+msgctxt "d1ce3c92-7025-4607-a910-444361a6b9b3/name:0"
+msgid "Red"
+msgstr "Roja"
```

Comment:
@nicpottier this test is showing what happens when we find two different translations of a base text